packages/embark-compiler/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "embark-compiler",
+  "version": "4.0.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark is a framework that allows you to easily develop and deploy DApps",
+  "homepage": "https://embark.status.im/",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "license": "MIT",
+  "repository": "github:embark-framework/embark",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --copy-files --extensions \".js,.ts\" --out-dir dist --source-maps",
+    "ci": "npm run qa",
+    "clean": "npx npm-run-all clean:* && npx rimraf .nyc_output coverage dist embark-*.tgz package",
+    "clean:templates": "node scripts/templates-reset",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint babel.config.js bin/embark src/bin/ src/lib/",
+    "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
```

Comment:
Copy `packages/embark/tslint.json` into `packages/embark-compiler/tslint.json` for this script to work.